gradle/dependencies.gradle

```diff
@@ -11,7 +11,7 @@ ext {
       mapboxMapSdk       : '6.1.3',
       mapboxSdkServices  : '3.1.0',
```

Comment:
Could you bump `mapboxSdkServices` to `3.2.0` 👀 https://github.com/mapbox/mapbox-java/releases/tag/v3.2.0